ex04.ex

```diff
@@ -55,7 +57,11 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  # Two helper functions to evaluate whether the minumum of two values
+  def min_helper(val1, val2) when val1 < val2, do: val1
+  def min_helper(val1, val2) when val1 >= val2, do: val2
```

Comment:
Technically you don't need the guard clause on the second form of the function...